src/Overrides/Dataset/UploadedFiles.tsx

```diff
@@ -69,20 +65,9 @@ export function UploadedFiles({ onChange, refresh }) {
             style={{ backgroundColor: 'white', border: '1px solid lightgrey', borderRadius: '4px', overflowY: 'auto', maxHeight: '400px' }}
           >
             <LoadingIndicatorView area={loadingArea} />
+
             {files.map((file) => (
-              <ListItem
-                key={file.id}
-                data-cy="uploaded-data-list-item"
-                button
-                onClick={() => {
-                  handleClick({
-                    display: file.name,
-                    path: file.id,
-                    type: DatasetType.Chem,
-                    uploaded: true, // indicates that file is already uploaded
-                  });
-                }}
-              >
+              <ListItemButton key={file.id} data-cy="uploaded-data-list-item" href={`?project=${file.id}`} component="a" target="_self">
```

Comment:
We probably want to have a slash in the href, i.e. `/?` instead of `?` @dvmoritzschoefl